src/main/io/serial_msp.c

```diff
@@ -1804,19 +1801,13 @@ static bool processInCommand(void)
         currentProfile->pidProfile.dterm_lpf_hz = read16();
         currentProfile->pidProfile.yaw_lpf_hz = read16();
         break;
-    case MSP_SET_ADVANCED_TUNING:
+    case MSP_SET_PID_ADVANCED:
         currentProfile->pidProfile.rollPitchItermIgnoreRate = read16();
         currentProfile->pidProfile.yawItermIgnoreRate = read16();
         currentProfile->pidProfile.yaw_p_limit = read16();
         currentProfile->pidProfile.deltaMethod = read8();
         masterConfig.batteryConfig.vbatPidCompensation = read8();
         break;
-    case MSP_SET_SPECIAL_PARAMETERS:
-        currentControlRateProfile->rcYawRate8 = read8();
-        masterConfig.rxConfig.airModeActivateThreshold = read16();
```

Comment:
You mean in the configurator? That's correct. We could still add it in case a future version of the configurator will be using it.